### DISCUSSION TITLE
Reanimated V2 for  React Native 0.72

### DISCUSSION THREAD

**Discussion by gift-sequoiaat** [software-mansion/react-native-reanimated#4693] [Q&A] [answered]
Is there a plan to provide reanimated V2 support for  React Native v 0.72

**Comment by tjzel** [comment]
Hi @gift-sequoiaat. Unfortunately at the moment we don't plan supporting React Native v0.72 in Reanimated2 and we strongly suggest upgrading to Reanimated3 if possible. Is anything blocking you from upgrading?

**Reply by gift-sequoiaat** [reply]
yeah @tjzel  i am in need of react native vision camera and i am using react native 0.72  i was  going through the draft pr of 1623 in vision camera  but its build.gradle seems to be failing  so i am unable to use it also is there any other alternatives of using Reanimated V2 with RN 0.72

**Reply by tjzel** [reply, marked as answer]
We are currently in progress of providing requested support for `react-native-vision-camera` in Reanimated3 so I suggest you stay tuned for it.

In the meantime you can probably contribute to fix that faulty build.gradle in vision camera's repo.

**Reply by gift-sequoiaat** [reply]
sure thanks @tjzel
